### DISCUSSION TITLE
MFA TOTP

### DISCUSSION THREAD

**Discussion by timsayshey** [in4it/vpn-server#24] [General]
Considering upgrading but I need MFA TOTP support. Is it something I can just run a script to add?

**Comment by wardviaene** [comment]
Hi Tim,

If you want to upgrade using the command line, you can use the commands listed at https://vpn-documentation.in4it.com/upgrade/

When you are on the latest version, you should be able to use MFA TOTP using the local database. If you're using OIDC/SAML, then you need to configure MFA on that level (your identity provider)
